prisma/migrations/migration_lock.toml

```diff
@@ -1,3 +1,3 @@
-# Please do not edit this file manually
+# Please do not edit this file manually y
```

Comment:
typo

Reply:
Resolved in b6d94d5a8fa385c07f09aa2db7c14964fd72d34d